packages/composables/src/getters/productGetters.ts

```diff
@@ -147,6 +150,23 @@ const getTotalReviews = (product: AgnosticProductVariant): number => {
 const getAverageRating = (product: AgnosticProductVariant): number => {
   return 0;
 };
+const getBreadcrumbs = (
+  product: AgnosticProductVariant
+): AgnosticBreadcrumb[] => {
+  if (!product.collections?.length) return [];
+  const collection = product?.collections?.slice(-1);
+
+  // TODO: Find a better solution for category path than hardcoding '/c/'
+  const breadcrumbs = collection[0]?.breadcrumbs?.map((breadcrumb) => ({
+    text: breadcrumb?.name === ROOT_COLLECTION ? 'Home' : breadcrumb?.name,
+    link: breadcrumb?.slug === ROOT_COLLECTION ? '/' : `/c/${breadcrumb?.slug}`
```

Comment:
I would avoid harcoding this kind of thing as you suggested. Have you tried adding `/c/` in theme instead? In theme there is a routes.js file that has whole routing defined so if we also define `/c/` rule there user would be able to adjust it to fit their needs but only if it will be part of theme. If it is a part of composables user cannot change that as it is a package from npm.

Reply:
I'll try that, thanks